src/applications/check-in/components/pages/demographics/DemographicsDisplay.jsx

```diff
@@ -20,42 +20,50 @@ export default function DemographicsDisplay({
   noAction = () => {},
   Footer,
 }) {
+  const { currentlyLoggedIn } = useSelector(state => state.user?.login);
+
   const dispatch = useDispatch();
   const setEditContext = useCallback(
     (data, url) => {
       dispatch(
         createSetEditContext({
           ...data,
-          originatingUrl: URLS.DEMOGRAPHICS,
-          editingPage: EDITING_PAGE_NAMES.DEMOGRAPHICS,
         }),
       );
       jumpToPage(url);
     },
     [dispatch, jumpToPage],
   );
+  const shared = {
```

Comment:
NOTE: This shared object is needed since we need to pass more data around when the link is clicked to build a good URL